docs/android.md

```diff
@@ -1,66 +1,49 @@
   - In Other Settings -> Configuration section -> choose targeted architectures
     <br><img src="images/android/selectArchitectures.png">
   - In Other Settings -> Configuration section -> select both Activity and GameActivity as Application Entry Point
     <br><img src="images/android/ApplicationEntryPoint.png" width='600px'>
-- Go back to Build Settings window
+- Go to Project tab and select MainApp under Assets / Plugins / Android.
+  - In Inspector tab -> Select platforms for plugin -> select Android
+  - In Inspector tab -> Platform settings -> Select dependent module -> choose None
+    <br><img src="images/android/androidlibImportSettings.png" width='700px'>
+- Go back to Build Profiles window.
   - Select option “Export Project” 
-    <br><img src="images/android/exportProject.png" width='400px'>
-  - Export UnityProject to androidBuild folder, the folder structure should look like this
-    <br><img src="images/android/exportedProjectFolder.png">
-    
-**3. Add Unity Library module to NativeAndroidApp**
-<br>Do the following to add the exported androidBuild/unityLibrary module to the NativeAndroidApp Gradle project in Android Studio:
-- Open NativeAndroidApp in Android Studio
-- Open settings.gradle file
-  - Add a new project pointing to unityLibrary module after the main app module
-    ```
-    include ':unityLibrary'
-    project(':unityLibrary').projectDir=new File('..\\UnityProject\\androidBuild\\unityLibrary')
-    ```
-  - And add the following in dependencyResolutionManagement{repositories{ block
-    ```
-    flatDir {
-      dirs "${project(':unityLibrary').projectDir}/libs"
-    }
-    ```
-    <img src="images/android/settingsGradle.png">
-- Open build.gradle(Module: NativeAndroidApp.app) file
-  - Add the following in dependencies{ block
-    ```
-    implementation project(':unityLibrary')
-    implementation fileTree(dir: project(':unityLibrary').getProjectDir().toString() + ('\\libs'), include: ['*.jar'])
-    ```
-    <img src="images/android/buildGradleApp.png">
-  
-  - In the same file take a look at android{defaultConfig{ndk{ block and make sure abiFilters match the architectures you selected in Unity editor before exporting the project. The filter must match architectures in Unity editor exactly. If Unity exports only ARMv7 architecture, but the filter includes arm64-v8a, the application will crash on ARM64 devices. Check for valid abiFilters values in the [official Android documentation](https://developer.android.com/ndk/guides/abis#sa).
+    <br><img src="images/android/exportProject.png" width='670px'>
+  - Export UnityProject to a folder, the folder structure should look like this. (If you see Multiple application entries pop-up, click Yes.)
+    <br><img src="images/android/exportedProjectFolder.png" width='250px'>
+
+**3. Prepare to build in Android Studio**
```

Comment:
There's not thing to prepare in Android Studio, user just needs to open it and build & run. Maybe let's remove this whole 3. block?

Reply:
I removed the step 3. 
Though, I thought that it would be good to provide information on where to compare or check when something goes wrong. :D